Assets/MRTK/Core/Utilities/CameraCache.cs

```diff
@@ -52,9 +52,16 @@ public static Camera Main
 
                 // Cache the main camera
                 cachedCamera = mainCamera;
-
                 return cachedCamera;
             }
         }
+
+        /// <summary>
+        /// Manually update the cached main camera 
+        /// </summary>
+        public static void UpdateCachedMainCamera(Camera camera)
```

Comment:
Looks like we used to have a `Refresh` method that was removed before we released:

![image](https://user-images.githubusercontent.com/3580640/105098514-078ce800-5a5f-11eb-8b20-e2d1c65a29ee.png)

I think this is a good re-addition! Thanks!